lib/taskwarrior-web/model/task.rb

```diff
@@ -136,6 +136,17 @@ def self.respond_to?(method_sym, include_private = false)
       end
     end
 
+    ################################################
+    # CLASS METHOD FOR SYNCRONISING WITH TASK SERVER
+    ################################################
+
+    ##
+    # Syncronise the local task database with the server
+    # TODO: These args are going to be ignored!
+    def self.sync(*args)
```

Comment:
Perhaps we should move this into the helpers.rb file (namespace TaskwarriorWeb::App::Helpers) as the method 'synchronise'?